apps/api/tests/test_admin.py

```diff
@@ -111,31 +112,80 @@ def test_can_retrieve_applicants(
 
 @patch("services.mongodb_handler.raw_update_one", autospec=True)
 @patch("services.mongodb_handler.retrieve_one", autospec=True)
-def test_can_submit_review(
+def test_can_submit_nonhacker_review(
     mock_mongodb_handler_retrieve_one: AsyncMock,
     mock_mongodb_handler_raw_update_one: AsyncMock,
 ) -> None:
-    """Test that a user can properly submit an applicant review."""
+    """Test that a user can properly submit a nonhacker applicant review."""
+    post_data = {"applicant": "edu.uci.sydnee", "score": 0}
+
+    returned_record: dict[str, Any] = {
+        "_id": "edu.uci.sydnee",
+        "roles": ["Applicant", "Mentor"],
+        "application_data": {
+            "reviews": [
+                [datetime(2023, 1, 19), "edu.uci.alicia", 100],
+            ]
+        },
+    }
 
-    mock_mongodb_handler_retrieve_one.return_value = REVIEWER_IDENTITY
+    mock_mongodb_handler_retrieve_one.side_effect = [
+        REVIEWER_IDENTITY,
+        returned_record,
+        REVIEWER_IDENTITY,
+        returned_record,
+    ]
```

Comment:
Note [for later]: this mocking is getting a bit ridiculous, I think it might be good to look into overriding the FastAPI dependency (think Sam and I did this in some other test file before).